src/main/java/org/schemaspy/util/JDot.java

```diff
@@ -0,0 +1,62 @@
+package org.schemaspy.util;
+
+import org.apache.commons.io.IOUtils;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.InputStream;
+
+public class JDot {
+
+    private static final String SIMULATE_VERSION = "2.26";
+    private static final String FK = "[FK] ";
+    private static final String PK = "[PK] ";
+    protected ScriptEngine scriptEngine;
+
+    public JDot() {
+        try {
+            InputStream vizJs = JDot.class.getResourceAsStream("/viz.js");
+            if (vizJs == null) {
+                throw new IllegalArgumentException("viz.js not found");
+            }
+            ScriptEngineManager scriptEngineManager = new ScriptEngineManager();
+            scriptEngine = scriptEngineManager.getEngineByName("JavaScript");
+            scriptEngine.eval(IOUtils.toString(vizJs,"UTF-8"));
+        } catch (Exception e) {
+            throw new IllegalArgumentException("viz.js", e);
+        }
+    }
+
+    protected String toSvg(String dotSource, int memSize) {
+        try {
+            scriptEngine.getBindings(ScriptContext.ENGINE_SCOPE).put("dotSource", dotSource);
+            scriptEngine.getBindings(ScriptContext.ENGINE_SCOPE).put("memSize", memSize);
+            return (String) scriptEngine.eval("Viz(dotSource,{ totalMemory: memSize })");
```

Comment:
```
return (String) scriptEngine.eval("Viz(dotSource, { totalMemory: memSize, images: [ { path: "images/primaryKeys.png" , width: "261px" , height: "261px" },{ path: "images/foreignKeys.png" , width: "261px" , height: "261px" }] })");
```

Something like that

Reply:
@npetzall  thanks for assisting! Could you please review latest code changes after your comments?